examples/cpython/minimqtt_simpletest_cpython.py

```diff
@@ -61,7 +63,7 @@ def message(client, topic, message):
 
 # Set up a MiniMQTT Client
 mqtt_client = MQTT.MQTT(
-    broker=os.getenv("broker"),
+    broker="io.adafruit.com",
```

Comment:
Maybe we stick with broker env var lookup and fall back to io.adafruit.com using the second parameter / default value, as this example isn't really IO related (just a good test server if the user already has an account). Similarly here: https://github.com/adafruit/Adafruit_CircuitPython_MiniMQTT/pull/245/files#diff-afdace9507c758aee40533c5ef2f1cacbfe152f00f601e2fc07571931dfa63f8R16 which is `examples/esp32spi/minimqtt_certificate_esp32spi.py` line 16.

Reply:
@tyeth , so do:
```
broker = getenv("broker", "io.adafruit.com")
```
in this one?

Reply:
yeah, something like that. think i spotted "broker" in a few other places too (just glacing at the connect lines)

Reply:
Will find all of these and make sure they are updated correctly. Will hold until I get other answers to do the work